### ISSUE TITLE
Not equal operator is not working with --treenode-filter command line parameter

### ISSUE THREAD

**Issue by sergey-rybalkin** [thomhurst/TUnit#1381]
Not sure whether this is a TUnit or TestFx issue, but it seems like not all of the test filtering operators documented in [Microsoft Testing Platform](https://github.com/microsoft/testfx/blob/main/docs/mstest-runner-graphqueryfiltering/graph-query-filtering.md) are supported with `--treenode-filter` command line parameter. In particular, not equal `!=` doesn't seem to be working for me. Consider the following code snippet:
```
public class MyTests
{
    [Test]
    [Category("Smoke")]
    public void TestWithCategory()
    {
        // ...
    }

    [Test]
    public void TestWithoutCategory()
    {
        // ...
    }
}
```

The following command runs only the first test, as expected `dotnet run --treenode-filter /*/*/*/*[Category=Smoke]`:

```
Test run summary: Passed! - bin\Debug\net9.0\Engine.Tests.dll (net9.0|x64)
  total: 1
  failed: 0
  succeeded: 1
  skipped: 0
  duration: 91ms
```
However, `dotnet run --treenode-filter /*/*/*/*[Category!=Smoke]` doesn't seem to find anything:
```
Test run summary: Zero tests ran - bin\Debug\net9.0\Engine.Tests.dll (net9.0|x64)
  total: 0
  failed: 0
  succeeded: 0
  skipped: 0
  duration: 88ms
```
**SDK version**: 9.0.101
**OS version**: Microsoft Windows [Version 10.0.26100.2454]
**TUnit version**: 0.4.86
**Microsoft.Testing.Platform version**: 1.4.3

**Comment by thomhurst** [comment]
Yeah this will be a Microsoft one - Let me see if I can transfer it to their repo

**Comment by thomhurst** [comment]
Ah I can't - I need write access to their repo to do that. Could you raise this on the testfx repo?

**Comment by sergey-rybalkin** [comment]
> Ah I can't - I need write access to their repo to do that. Could you raise this on the testfx repo?

Sure, thanks

**Comment by sergey-rybalkin** [comment]
Seems like it can be closed now as it was fixed in Microsoft.Testing.Platform v1.6
